app/nyu-devops-s17-inventory/static/index.html

```diff
@@ -1,24 +1,43 @@
 <!DOCTYPE html>
 <html>
-
   <head>
-    <title>Python Flask Starter Application</title>
+    <title>Inventory REST API Service</title>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="static/stylesheets/style.css">
   </head>
-
   <body>
-    <table>
-      <tr>
-        <td style= "width:30%;">
-          <img class = "newappIcon" src="static/images/newapp-icon.png">
-        <td>
-            <h1>Hi there!</h1>
-          <p>Thanks for creating a
-            <span class = "blue">Python <a href = "http://flask.pocoo.org/">Flask</a> Starter Application</span>.
-    </table>
+      <table>
+    <tr>
+            <td style= "width:30%;">
+              <img class = "newappIcon" src="static/images/newapp-icon.png">
+            </td>
+      <td>
+        <h1 id = "message">Inventory REST API Service</h1>
+        <p class='description'>
+        GET <a href="/inventory/products">/inventory/products</a> to see all the products.
+        <br><br>
+        GET /inventory/products/{id} retrieves a product from the DB using an ID.
+        <br><br>
+        PUT /inventory/products/{id} updates a product using an ID.
```

Comment:
update method only updates the quantity of a single type for a kind of product :
{
       "type":"open_box", 
        "quantity":20
}